### ISSUE TITLE
Build output with tsconfig.json is relative to file on which the build was executed.

### ISSUE THREAD

**Issue by mconner** [Microsoft/TypeScript-Sublime-Plugin#414]
When I try to build (Ctrl-B), it has to be executed on a typescript file (e.g.: myfile.ts,  not tsconfig.json).  If I do that, with a tsconfig.json like:
{
    "compilerOptions": {
        "noImplicitAny": false,
        "preserveConstEnums": true,
        "out": "build/tsc.js",
        "sourceMap": true
    }
 }
and invoke a build when myfile.ts is at the root of the project, it works.  However, if I build from a file, at foo/bar/somefile.ts, then the output is at foo/bar/build/tsc.js.  I would think it should be relative to the root of the project (as defined by tsconfig.json), and not whatever file I happen to have clicked on to build it.  
Also, it seems like it should be possible to build when the tsconfig.json, itself, is selected.

**Comment by mhegazy** [comment]
Set "rootDir" : "."

**Comment by mconner** [comment]
@mhegazy  I tried that.  I added rootDir, as you show, save, close sublime (to make sure the change was recognized), reopen it, click on one of the .ts files in a folder, and I get  the same results:  It builds the output relative to the file I had selected when I initiated the build, not the root of the project.  The docs indicate that rootDir is "Only use to control the output directory structure with --outDir.", which I am not using.  

As I'm writing this, and looking at the docs, I find that out is deprecated.  I tried outFile, and it appears to work correctly, with or without rootDir, so it appears that outDir is not a direct replacement for out.
